app/src/main/java/com/example/dine_and_donate/Activities/HomeActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.example.dine_and_donate.Activities;
+
+import android.os.Bundle;
+import android.view.MenuItem;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+
+import com.example.dine_and_donate.HomeFragments.MapFragment;
+import com.example.dine_and_donate.HomeFragments.NotificationsFragment;
+import com.example.dine_and_donate.HomeFragments.ProfileFragment;
+import com.example.dine_and_donate.R;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+
+public class HomeActivity extends AppCompatActivity {
+
+    private BottomNavigationView bottomNavigationView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home);
+
+        final FragmentManager fragmentManager = getSupportFragmentManager();
+
+        fragmentManager.beginTransaction().replace(R.id.flContainer, new ProfileFragment()).commit();
+
+        bottomNavigationView = findViewById(R.id.bottom_navigation);
+
+        bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                Fragment fragment = new ProfileFragment();
```

Comment:
@ryan-beck i think this is your code? 

it seems like you're creating this unnecessarily, just set it to null and then it will get set in the switch statement below. otherwise you're creating a bunch of extra ProfileFragments